packages/netlify-cms-backend-git-gateway/src/implementation.ts

```diff
@@ -125,18 +152,26 @@ export default class GitGateway implements Implementation {
       this.backendType = null;
     }
 
-    this.authClient = window.netlifyIdentity
-      ? window.netlifyIdentity.gotrue
```

Comment:
The `this.authClient` is mainly used by the normal ` gotrue-js` mode, how about we skip setting the `window.netlifyIdentity.gotrue` for identity widget here ?

Reply:
Not sure I understand what "normal `gotrue-js` mode" means.

Reply:
If you use the `git-gateway` backend without including the identity widget via a module of script tag, you will be greeted with a simple email and password input form on login that is using the `gotrue-js` module directly to authenticate users. My point is the `this.authClient` that is set here is mainly used by this mode to login [here](https://github.com/netlify/netlify-cms/blob/master/packages/netlify-cms-backend-git-gateway/src/AuthenticationPage.js#L146)  logout [here](https://github.com/netlify/netlify-cms/blob/master/packages/netlify-cms-backend-git-gateway/src/implementation.ts#L249)  but over [here](https://github.com/netlify/netlify-cms/blob/master/packages/netlify-cms-backend-git-gateway/src/implementation.ts#L237) we can do an `if` check, thoughts?

Reply:
Thanks I get it now, good point. Let me clean up the code and test again

Reply:
We should still wait for the widget to initialise before calling any of its methods though.